heutagogy/views.py

```diff
@@ -196,6 +199,10 @@ def post(self, id):
                 bookmark.read = aniso8601.parse_datetime(update['read'])
             else:
                 bookmark.read = None
+
+        if 'meta' in update:
+            bookmark.meta = update['meta'] if update['meta'] else None
```

Comment:
We already know that `meta` is present in `update`, so what's the point of this `if`? `bookmark.meta = update['meta']` would work just fine.